_data/members.yml

```diff
@@ -136,11 +136,11 @@
 
 - name:  Sebastian Prillo
   degrees:
-  image: john_d.jpg
+  image: sebastian_prillo.jpg
   altimage:
   position: Graduate student (EECS); joint with Yun Song
```

Comment:
should it be Yun S. Song?

Reply:
Ah, good point, I think what you suggest is better. Thanks! I'll update this.